maestro-test/src/test/resources/095_launch_arguments.yaml

```diff
@@ -0,0 +1,11 @@
+appId: com.example.app
+env:
+  ARGUMENT_B: argumentB
+  INCLUDE_ARGUMENT_C: true
+---
+- launchApp:
+    arguments:
+      - argumentA
+      - ${ARGUMENT_B}
+      - ${INCLUDE_ARGUMENT_C && 'argumentC'}
+      - ${INCLUDE_ARGUMENT_D && 'argumentD'}
```

Comment:
Slightly confused with the arguments here. Can it be an independent variable or key-value pairs only?

Reply:
It can be independent variable (that is what iOS typically expects). In Android terms that could be either a value (in which case it will be passed as a Boolean flag) or key-value (in which case it will be passed as a String)

Reply:
How would android behave if an argument only is supplied will it ignore?

Reply:
Curious how iOS reads a single argument. @artem888 any idea how would single argument (argumentA) be read in iOS apps?